ansible_collections/arista/cvp/plugins/module_utils/resources/schemas/v3.py

```diff
@@ -755,3 +755,101 @@
         }
     }
 }
+
+SCHEMA_CV_VALIDATE = {
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "$id": "http://example.com/example.json",
+    "type": "array",
+    "default": [],
+    "title": "Root Schema",
+    "additionalProperties": False,
+    "items": {
+        "type": "object",
+        "default": {},
+        "title": "A Schema",
+        "required": ["device_name", "search_type", "local_configlets", "cvp_configlets"],
```

Comment:
```suggestion
        "required": ["device_name", "search_type"],
```
We don't want both `local_configlets` and `cvp_configlets` to be required.

Reply:
agreed from my end.